spec/models/activity_spec.rb

```diff
@@ -652,4 +652,26 @@
       expect(project.planned_disbursements_total).to eq(1570.00)
     end
   end
+
+  describe "#actual_total_for_report" do
+    it "returns the total of all the activity's transactions scoped to a report" do
+      project = create(:project_activity, :with_report)
+      report = Report.find_by(fund: project.associated_fund, organisation: project.organisation)
+      _transaction_1 = create(:transaction, parent_activity: project, value: 100.20, report: report, date: Date.today)
```

Comment:
Do these values need to be assigned to variables? If the variable is unused we could remove the assignment entirely.

Reply:
They don't! Old habit 😁